fileglancer/handlers.py

```diff
@@ -24,6 +24,17 @@ def _get_mounted_filestore(fsp):
     return filestore
 
 
+def get_current_user(api):
+    """
+    Get the current user's username.
+    
+    Returns:
+        str: The username of the current user.
+    """
+    #return api.current_user.name
+    return os.getenv("USER") or os.getenv("USERNAME")
+
```

Comment:
What about subclassing APIHandler, adding this as a class method and then falling back to self.current_user.name if the os.getenv() functions fail to return a value?

Reply:
How delightfully object oriented! Fixed.